td.vue/src/service/ajv-SchemaVerification.js

```diff
@@ -0,0 +1,17 @@
+import Ajv from 'ajv';
+import { schema } from '../assets/threat-model-schema';
+
+export const isValidSchema = (jsonFile) => {
+
+    const ajv = new Ajv({strict : false});
+    const validate = ajv.compile(schema);
```

Comment:
compile of the schema is relatively slow, so could this be done during the application initialization, according to https://ajv.js.org/guide/managing-schemas.html#compiling-during-initialization